tools/htmlpurifier/HTMLPurifier.standalone.php

```diff
@@ -332,8 +333,8 @@ public static function arborize($tokens, $config, $context) {
             if ($token instanceof HTMLPurifier_Token_End) {
                 $token->start = null; // [MUT]
                 $r = array_pop($stack);
-                assert($r->name === $token->name);
-                assert(empty($token->attr));
+                //assert($r->name === $token->name);
```

Comment:
I'm worried about commented code: we use the real version of 4.10 or we don't use it but there is no way we "hack" it. Also why not use Composer to do that instead? https://packagist.org/packages/ezyang/htmlpurifier

Reply:
The code is commented by HTMLPurifier in the original 4.10.0.

Reply:
Sad !

![giphy](https://user-images.githubusercontent.com/1247388/45367719-2dc63180-b5e2-11e8-9830-d29403c0f358.gif)

Reply:
no, it's not your fault, it's HTMLPurifier's to do it ^^